src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -123,6 +138,45 @@ public void Load(string settingsXml)
 #endif
         }
 
+        public void SaveRandomSeed(string dirname)
+        {
+            TextWriter writer = null;
+
+            try
+            {
+                writer = new StreamWriter(Path.Combine(dirname, "$RANDOM_SEED$"));
+                writer.Write(RandomSeed.Value);
```

Comment:
Do we need to worry about non-english cultures here? Should we be writing this out as invariant culture and parsing it below as invariant? Probably not since it is the same machine right?